cli/src/action/api.rs

```diff
@@ -65,8 +65,15 @@ pub struct SplinterRestClient {
 impl SplinterRestClient {
     /// Gets the Splinter node's status.
     pub fn get_node_status(&self) -> Result<NodeStatus, CliError> {
-        Client::new()
-            .get(&format!("{}/status", self.url))
+        #[allow(unused_mut)]
+        let mut request = Client::new().get(&format!("{}/status", self.url));
+
+        #[cfg(feature = "splinter-cli-jwt")]
+        {
+            request = request.header("Authorization", format!("Bearer Cylinder:{}", &self.token));
```

Comment:
I think it would be a better design to have the client take the entire authorization string (`Bearer Cylinder:<token>`) instead of just the token. The client shouldn't be restricted to only one type of authorization; in the future, we may decide it should support some other kind of authorization as well.

I'd suggest updating the `create_token` method to be something like `create_cylinder_jwt_auth` where it returns the whole authorization string, since the token is only really useful as part of the auth string.

Reply:
Changed client to have a `jwt-auth` field instead of a token field which will hold the full authorization string. Changed create_token to create_cylinder_jwt_auth and modified it to build the whole authorization string